### DISCUSSION TITLE
Web page been cached

### DISCUSSION THREAD

**Discussion by markdem** [SafeExamBrowser/seb-win-refactoring#774] [General]
WIN11
SEB 3.6.0

I have a landing page that is been delivered form a local web server that acts as the start page inside SEB. I have updated a URL in this page. On some (about 40%) of my exam laptops SEB still shows the old page.

I have checked the config, the "delete cache" option on on the browser tab is enabled and both option in the "Session Handling" are enabled.
If I open the URL in edge the new page will open.
If I logon as a different user on the same computer the new page will open. 
I have tried to uninstall and reinstall SEB, did not help.

Does anyone know where the actual cache is stored, or is there anything else I can try to get this working?

Thanks

**Comment by markdem** [comment]
Just a update, if I allow the dev tools to be opened and select "Disable Cache" under the Network tab in the dev window, the new page will be loaded if I click refresh in SEB. The new page will now load correctly even after the dev tools have been closed and SEB restarted.

**Reply by dbuechel** [reply]
The browser cache is located under `%LocalAppData%\SafeExamBrowser`. With respect to the caching issue itself, you'd need to create a bug report specifying a complete and reliable way to reproduce the problem if we were to investigate the issue further.
